compiler/noirc_evaluator/src/ssa/opt/unrolling.rs

```diff
@@ -151,7 +151,10 @@ impl Loops {
             if next_loop.blocks.iter().any(|block| self.modified_blocks.contains(block)) {
                 let mut new_context = find_all_loops(function);
                 new_context.failed_to_unroll = self.failed_to_unroll;
-                return new_context.unroll_each_loop(function);
+                return unroll_errors
```

Comment:
Found this bug in aztec-packages when trying to use the new slice capabilities :)

Reply:
Like this is the fix?